hackathon/views.py

```diff
@@ -1,13 +1,13 @@
 from datetime import datetime
-
 from django.views.generic import ListView
 from django.shortcuts import render, get_object_or_404, redirect
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
-
-from .models import Hackathon
+from django.utils import timezone
+from .models import Hackathon, HackTeam, HackProject, HackProjectScore, HackProjectScoreCategory
```

Comment:
There should be a blank line between 3rd party and local modules.

Reply:
done